Source/Scene/Billboard.js

```diff
@@ -1109,12 +1109,22 @@ Billboard._updateClamping = function (collection, owner) {
         clampedPosition.x += position.height;
       }
     }
-    owner._clampedPosition = Cartesian3.clone(
-      clampedPosition,
-      owner._clampedPosition
-    );
+
+    // Billboards can be reused, i.e. by EntityCluster so check that the height
```

Comment:
For my understanding, is this also intended to fix the bug from your [comment](https://github.com/CesiumGS/cesium/issues/8006#issuecomment-667301460) on the issue?

> It looks like the QuadtreePrimitive customData list keeps growing and some stale height callbacks are being made with old coordinates.

I don't think the comment in the code is needed.